CMakeLists.txt

```diff
@@ -215,6 +215,7 @@ endif()
 # HIP is always required
 find_package(hip REQUIRED PATHS /opt/rocm)
 message(STATUS "Build with HIP ${hip_VERSION}")
+option(APPEND_INTERFACE_INCLUDE_DIRECTORIES "Append include directories to compiler flags" ON)
```

Comment:
```suggestion
option(MIOPEN_HIP_COMPILER_USE_SYSTEM_INCLUDE_DIRECTORIES "Append include directories to compiler flags" ON)
```

Reply:
Updated the patch with the suggested name